dist/index.html

```diff
@@ -17,8 +17,14 @@
   </header>
   <main>
     <section class="hero">
-      <article class="login-screen hidden" id="loginSection">
-        
+      <article class="login-screen" id="loginSection">
```

Comment:
Love your naming convention. Everything is easy to identify and anyone who looks at your project will know what they are looking at.